packages/@react-aria/menu/docs/useMenu.mdx

```diff
@@ -128,7 +128,7 @@ import {mergeProps} from '@react-aria/utils';
 
 function Menu(props) {
   // Create state based on the incoming props
-  let state = useTreeState({...props, selectionMode: 'none'});
+  let state = useTreeState({...props, disabledKeys:['one'], selectionMode: 'none'});
```

Comment:
Just a small suggestion: I think this `disabledKeys` should be passed in as one of the Menu props like so
```
<Menu onAction={alert} aria-label="Actions" disabledKeys={["one"]}>
  <Item key="one">One</Item>
  <Item key="two">Two</Item>
  <Item key="three">Three</Item>
</Menu>
```

Reply:
yeah, better will change it.